packages/ignite-dev-screens/templates/PluginExamplesScreen.js

```diff
@@ -1,14 +1,19 @@
 // @flow
 
-import React, { PropTypes } from 'react'
+// Fair Warning:  PluginExamples has a good bit of Ignite automation in editing.
```

Comment:
I'm not sure this comment needs to be here.  I think this comment rightfully applies to every single template we hand off to the end-user.